rita_client/src/operator_update/ops_websocket.rs

```diff
@@ -79,8 +113,13 @@ pub fn send_websocket_update() {
                                         // tht will then fall out of this loop into the outer loop, restarting the whole thing
                                         // and reconnecting the socket
                                         let msg = msg.unwrap();
-                                        if let Some(hour) = handle_received_operator_message(msg) {
-                                            ops_last_seen_usage_hour = Some(hour);
+                                        if let Some(hour) = handle_received_operator_message(msg, &our_secretkey, Some(&ops_pubkey)) {
+                                            match hour {
+                                                ReceivedOpsData::UsageHour(hour) => {
+                                                    ops_last_seen_usage_hour = Some(hour);
+                                                },
+                                                _ => panic!("Why are we receiving a public key again from ops? restarting!"),
```

Comment:
we should probably handle a repeat sending gracefully, if ops gets restarted it will send it again right?

Reply:
Yes, the idea here is that if ops restarts (which will cut off the websocket connection regardless) we'll get a new key, but the only time we send keys on the ops side are when a connection is first opened. on the router side if this thread panics we restart that loop and get a new connection

Reply:
Edited so now we gracefully change the ops server key